ShareTheTable/View Controllers/SearchViewController.m

```diff
@@ -0,0 +1,103 @@
+//
+//  SearchViewController.m
+//  ShareTheTable
+//
+//  Created by Andrea Gonzalez on 7/12/22.
+//
+
+#import "SearchViewController.h"
+#import <QuartzCore/QuartzCore.h>
+#import "Parse/Parse.h"
+#import "ResultsViewController.h"
+
+static CGFloat const kViewControllerRangeSliderWidth = 290.0;
+static CGFloat const kViewControllerLabelWidth = 100.0;
+
+@interface SearchViewController ()
+
+@end
+
+@implementation SearchViewController
+
+-(void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender{
+     if([segue.identifier isEqualToString:@"searchToResultsSegue"]){
+         ResultsViewController* controller = [segue destinationViewController];
+         controller.minAge = @([self.rangeSlider minimumValue]);
+         controller.maxAge = @([self.rangeSlider maximumValue]);
+     }
+ }
+
+- (IBAction)didTapFindUsers:(id)sender {
+    [self performSegueWithIdentifier:@"searchToResultsSegue" sender:sender];
+}
+
+- (IBAction)didTapDietaryRestrictions:(id)sender {
+}
+
+- (void)viewDidLoad {
```

Comment:
Try to order methods in the order you call them, within their sections. viewDidLoad should be first, prepareForSegue is usually towards the end. Improves readability and makes it easier for you to keep track of your work!